pkgs/applications/audio/dsf2flac/default.nix

```diff
@@ -2,8 +2,8 @@
 , taglib, zlib }:
 
 stdenv.mkDerivation rec {
-  pname = "dsf2flac";
-  version = "unstable-2018-01-02";
+  pname = "dsf2flac-unstable";
```

Comment:
this will break the build, as pname is used to determine the src

Reply:
thanks. fixed